network-errors/src/commonMain/kotlin/dev/icerock/moko/network/errors/NetworkExceptionMappers.kt

```diff
@@ -52,27 +54,53 @@ fun ExceptionMappersStorage.registerAllNetworkMappers(
  */
 private fun getNetworkErrorExceptionStringDescMapper(
     errorException: ErrorException,
-    unauthorizedErrorText: StringResource,
-    notFoundErrorText: StringResource,
-    accessDeniedErrorText: StringResource,
-    internalServerErrorText: StringResource
+    httpNetworkErrorsTexts: HttpNetworkErrorsTexts
 ): StringDesc {
     val httpStatusCode = errorException.httpStatusCode
     return when {
-        errorException.isUnauthorized -> unauthorizedErrorText.desc()
-        errorException.isNotFound -> notFoundErrorText.desc()
-        errorException.isAccessDenied -> accessDeniedErrorText.desc()
+        errorException.isUnauthorized -> httpNetworkErrorsTexts.unauthorizedErrorText.desc()
+        errorException.isNotFound -> httpNetworkErrorsTexts.notFoundErrorText.desc()
+        errorException.isAccessDenied -> httpNetworkErrorsTexts.accessDeniedErrorText.desc()
         httpStatusCode >= HttpStatusCode.InternalServerError.value &&
                 httpStatusCode <= INTERNAL_SERVER_ERROR_CODE_MAX -> {
             StringDesc.ResourceFormatted(
-                internalServerErrorText,
+                httpNetworkErrorsTexts.internalServerErrorText,
                 httpStatusCode
             )
         }
         else -> errorException.description?.desc() ?: ExceptionMappersStorage.getFallbackValue()
     }
 }
 
+/**
+ * Maps the input error [sslException] to ssl error text.
+ */
+@Suppress("ComplexMethod")
+private fun getSSLExceptionStringDescMapper(
+    sslException: Throwable,
+    sslNetworkErrorsTexts: SSLNetworkErrorsTexts
+): StringDesc {
+    return when (sslException.getSSLExceptionType()) {
+        SSLExceptionType.SecureConnectionFailed -> (sslNetworkErrorsTexts.secureConnectionFailed
+            ?: sslNetworkErrorsTexts.default).desc()
+        SSLExceptionType.ServerCertificateHasBadDate -> (sslNetworkErrorsTexts.serverCertificateHasBadDate
+            ?: sslNetworkErrorsTexts.default).desc()
+        SSLExceptionType.ServerCertificateUntrusted -> (sslNetworkErrorsTexts.serverCertificateUntrusted
+            ?: sslNetworkErrorsTexts.default).desc()
+        SSLExceptionType.ServerCertificateHasUnknownRoot -> (sslNetworkErrorsTexts.serverCertificateHasUnknownRoot
+            ?: sslNetworkErrorsTexts.default).desc()
+        SSLExceptionType.ServerCertificateNotYetValid -> (sslNetworkErrorsTexts.serverCertificateNotYetValid
+            ?: sslNetworkErrorsTexts.default).desc()
+        SSLExceptionType.ClientCertificateRejected -> (sslNetworkErrorsTexts.clientCertificateRejected
+            ?: sslNetworkErrorsTexts.default).desc()
+        SSLExceptionType.ClientCertificateRequired -> (sslNetworkErrorsTexts.clientCertificateRequired
+            ?: sslNetworkErrorsTexts.default).desc()
+        SSLExceptionType.CannotLoadFromNetwork -> (sslNetworkErrorsTexts.cannotLoadFromNetwork
+            ?: sslNetworkErrorsTexts.default).desc()
+        else -> ExceptionMappersStorage.getFallbackValue()
+    }
+}
```

Comment:
we should simplify method.
for example we can extract function for this pattern: `(sslNetworkErrorsTexts.secureConnectionFailed
            ?: sslNetworkErrorsTexts.default).desc()`